### PR TITLE
Avoid TypeError in ZmComposeView.prototype.enableInputs

### PR DESCRIPTION
**Pull request by barrydegraaff**
Using typeof does not avoid the error. @silentsakky perhaps you can merge it.

### CODE REVIEW

File: WebRoot/js/zimbraMail/mail/view/ZmComposeView.js [hunk 1]
```diff
@@ -1439,9 +1439,9 @@ ZmComposeView.prototype.enableInputs =
 function(bEnable) {
     DBG.println('draft', 'ZmComposeView.enableInputs for ' + this._view + ': ' + bEnable);
     this._recipients.enableInputs(bEnable);
-    if (typeof(this._subjectField) !== 'undefined') {
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/mail/view/ZmComposeView.js]: not exactly sure, we should be doing try/catch
can you check if below fix helps?
`if (typeof(this._subjectField) !== 'undefined' && typeof(this._bodyField) !== 'undefined') {`

**Comment by barrydegraaff** [comment]
I am testing your suggestion, as the bug does not happen every day 
it takes me a week or 2 before I can tell if it works. 

The try/catch did resolve the error. Then other errors show up, but those seem 
to not break anything and are only logged via AjxException. 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0

**Comment by barrydegraaff** [comment]
Unfortunately the problem I am having, returned when using 
if (typeof(this._subjectField) !== 'undefined' && typeof(this._bodyField) !== 'undefined') 
instead of try-catch. 

The problem is that when I click new mail, sometimes I need to click it twice to make 
it work because of this issue. 

Please consider merging the PR. 

Kind regards, 

Barry de Graaff 
Zeta Alliance 
Co-founder & Developer 
zetalliance.org | github.com/Zimbra-Community 

Signal: +31 617 220 227 
Fingerprint: 97f4694a1d9aedad012533db725ddd156d36a2d0

**Comment by barrydegraaff** [comment]
@Prashantsurana this one seems to be stuck. Maybe a try...catch is not the best resolution, but I cannot seem to make it work with typeOf and if.

**Pull request merged**